ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -406,6 +407,7 @@ public void setAllowFontScaling(ReactEditText view, boolean allowFontScaling) {
   @ReactProp(name = "placeholder")
   public void setPlaceholder(ReactEditText view, @Nullable String placeholder) {
     view.setHint(placeholder);
+    view.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
how will this work in a multiline TextInput? Thanks a lot for the PR :pray: :peace_symbol:

Reply:
I haven't tried it yet, will do next week

Reply:
Have you tried it yet?